forms/views/admin/builder.php

```diff
@@ -173,7 +173,7 @@ function htmlDecodeAll($str)
         <?php if (!empty($notices))
         {
             ?>
-            <div id="message" class="updated notice notice-success is-dismissible"><p><?php echo filter_var($notices, FILTER_SANITIZE_STRING); ?></p></div>
+            <div id="message" class="updated notice notice-success is-dismissible"><p><?php echo $notices; ?></p></div>

```

Comment:
It seems that $notices does not relate to the client array, do we know why we are removing the FILTER_SANITIZE_STRING here?

Reply:
As discussed, leaving it back to original form for now as we aren't sure how it was used.